core/src/main/scala/runtime/metrics.scala

```diff
@@ -26,14 +26,25 @@ import freestyle.opscenter.Models.{Metric, Metrics, MetricsList}
 import _root_.fs2._
 import org.http4s.websocket.WebsocketBits.{Binary, Text}
 import org.http4s.websocket.WebsocketBits.WebSocketFrame
-import java.time.Instant
+import shapeless.Lazy
```

Comment:
I guess you are not using this here, right?

Reply:
Sorry, I was using Lazy but definitively it was not neccesary and I remove it.